pgp/gnupg.go

```diff
@@ -121,19 +166,43 @@ func (g *GpgSigner) ClearSign(source string, destination string) error {
 
 // GpgVerifier is implementation of Verifier interface using gpgv as external program
 type GpgVerifier struct {
+	gpg      string
+	gpgv     string
 	keyRings []string
 }
 
+// NewGpgVerifier creates a new gpg signer
+func NewGpgVerifier() *GpgVerifier {
+	gpg, err := findGPG1()
+	if err != nil {
+		panic(err)
+	}
+
+	gpgv, err := findGPGV1()
+	if err != nil {
+		panic(err)
+	}
+
+	return &GpgVerifier{gpg: gpg, gpgv: gpgv}
+}
+
 // InitKeyring verifies that gpg is installed and some keys are trusted
 func (g *GpgVerifier) InitKeyring() error {
-	err := exec.Command("gpgv", "--version").Run()
+	cmd, err := findGPG1()
 	if err != nil {
-		return fmt.Errorf("unable to execute gpgv: %s (is gpg installed?)", err)
+		return err
+	}
+	g.gpg = cmd
```

Comment:
what is the reason that g.gpg and below g.gpgv is reinitialized again here? Is this not already done in NewGpgVerifier?